packages/peregrine/lib/talons/OrderHistoryPage/orderHistoryContext.js

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useContext, useMemo } from 'react';
+
+import DEFAULT_OPERATIONS from './orderHistoryContext.gql';
+import { useQuery } from '@apollo/client';
+
+const OrderHistoryContext = createContext();
+
+export const OrderHistoryContextProvider = props => {
+    const operations = Object.assign({}, DEFAULT_OPERATIONS, props.operations);
+    const { getProductURLSuffixQuery } = operations;
+
+    const { data } = useQuery(getProductURLSuffixQuery, {
+        fetchPolicy: 'cache-and-network'
```

Comment:
Since there are no `storeConfig` mutations, I didn't think there would be a scenario where this would change, so felt the real `cache-and-network` was appropriate here and wouldn't lead to unnecessary re-fetches. I'm not 100% on that though, and would like clarification on our desired fetch policies; I had assumed we were only leveraging `nextFetchPolicy: 'cache-first'` to be backwards compatible during the v3 upgrade, and intended to use `cache-and-network` as designed at some point.

Reply:
You're correct that we don't expect mutations to affect `storeConfig` however I believe the `cache-and-network` policy alone will re-query the network if the cached object changes at all.

One example might be this query just gets `storeConfig.product_url_suffix` while another gets `storeConfig.store_name`. I believe the addition of `store_name` to the cached object would trigger a requery unless you have `nextFetchPolicy: cache-first`. Would need to double check.

This is really only an issue while the component is mounted, so if you're not on the order history page it won't matter. Likely not a big deal and I wouldn't spend too much time on it.